src/app/pages/about/about.component.html

```diff
@@ -25,251 +25,48 @@ <h1 class="mb-3" id="title">About Us</h1>
-          Click here to sign Tommy's virtrolio.
-        </a></p>
-      </div>
-
-      <!-- Tommy's Contact Info. -->
-      <div class="col-md-3 order-md-1">
-        <p><strong>Contact Tommy:</strong></p>
-        <ul>
-          <li>LinkedIn: <a href="https://www.linkedin.com/in/tommyhuang999" target="_blank">Tommy Huang</a></li>
-          <li>Github: <a href="https://github.com/TommyHuang-dev" target="_blank">TommyHuang-dev</a></li>
-          <li>Website: <a href="https://tommyhuang.net/" target="_blank">tommyhuang.net</a></li>
-        </ul>
-      </div>
-
-    </div>
-    <hr/>
-
-    <!-- Janakitti's Row-->
-    <div class="row d-flex">
-
-      <!-- Janakitti's Image -->
-      <div class="col-md-2 order-md-2 image-container">
-        <img alt="Picture of Janakitti Ratana-Rueangsri" class="img-fluid regular-picture"
-             src="{{ Janakitti.regularLink }}">
-        <img alt="Silly picture of Janakitti Ratana-Rueangsri" class="img-fluid silly-picture"
-             src="{{ Janakitti.sillyLink }}">
-      </div>
-
-      <!-- Janakitti's Blurb -->
-      <div class="col-md-7 order-md-3 mt-3 mt-md-0">
-        <p class="text-justify">Janakitti is the Gandalf of our team at Virtrolio. While a true technical and creative
-          wizard on the keyboard and on Powerpoint, Janakitti is more importantly the rope that ties the sac together.
-          He is currently studying Software Engineering at the University of Waterloo. In his spare time, Janakitti is
-          known to be an extremely talented guitar player! You should also check out his extravagant website at
-          <a href="http://janakitti.com" target="_blank">janakitti.com</a></p>
-        <p><a href="/signing?uid=hTYLpcLKneOLsonAX5ZirLphc6h2&key=wizard">
-          Click here to sign Janakitti's virtrolio.
-        </a></p>
-      </div>
-
-      <!-- Janakitti's Contact Info. -->
-      <div class="col-md-3 order-md-1">
-        <p><strong>Contact Janakitti:</strong></p>
-        <ul>
-          <li>LinkedIn: <a href="https://www.linkedin.com/in/janakittiratana/" target="_blank">Janakitti
-            Ratana-Rueangsri</a></li>
-          <li>Github: <a href="https://github.com/janakitti" target="_blank">Janakitti</a></li>
-          <li>Instagram: <a href="https://www.instagram.com/janakitti/" target="_blank">@janakitti</a></li>
-          <li>Website: <a href="http://janakitti.com" target="_blank">janakitti.com</a></li>
-        </ul>
-      </div>
-
+    <!-- Each Row-->
+    <div *ngFor="let contributor of contributors">
+      <div class="row d-flex my-2">
+
+        <!-- Each Image -->
+        <div class="col-md-2 order-md-2 image-container vertical-center">
+          <img alt="Picture of {{contributor.name}}" class="img-fluid regular-picture" src="{{ contributor.images.regular }}">
+          <img alt="Silly picture of {{contributor.name}}" class="img-fluid silly-picture" src="{{ contributor.images.silly }}">
```

Comment:
You can't add spaces in one tag and leave them out in the next xD
```suggestion
          <img alt="Picture of {{ contributor.name }}" class="img-fluid regular-picture" src="{{ contributor.images.regular }}">
          <img alt="Silly picture of {{ contributor.name }}" class="img-fluid silly-picture" src="{{ contributor.images.silly }}">
```